account_move_name_sequence/tests/test_account_move_name_seq.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2021 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# @author: Moisés López <moylop260@vauxoo.com>
+# @author: Francisco Luna <fluna@vauxoo.com>
```

Comment:
I'd remove these headers altogeter. This is not standard and it's redundant, as the README already contains all involved people. Also from views

Reply:
It is already used from original module by original author

Also, this is not a change of migration to 15.0 but 14.0 then FW to 15.0